test/ufuzz.js

```diff
@@ -135,23 +182,25 @@ var FUNC_TOSTRING = [
     "}();",
     ""
 ].join("\n");
+var loops = 0;
+var funcs = 0;
 
 function run_code(code) {
     var stdout = "";
     var original_write = process.stdout.write;
-    process.stdout.write = function(chunk) {
+    process.stdout.write = function (chunk) {
```

Comment:
While I appreciate the updating of two-space to four-space indentation for consistency, can we avoid these extraneous whitespaces getting inserted in the process?